cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -11,18 +13,23 @@ var EventObserver = require( '../modules/util/EventObserver' );
  *
  * @param {HTMLNode} element
```

Comment:
Why do you need the `toggle` and `expandBegin/collapseBegin` events? Could you consolidate the events?